pages/docs/tooling/tooling-performance-analysis.md

```diff
@@ -1,70 +1,283 @@
+                                          advance |  183083.0        1  183083.0  183083.0  183083.0
+                          advance/m2n.receiveData |    1362.0        1    1362.0    1362.0    1362.0
+       advance/map.bbm.mapData.FromA-MeshToB-Mesh |      86.0        1      86.0      86.0      86.0
+advance/map.vci.computeMapping.FromA-MeshToB-Mesh |  181377.0        1  181377.0  181377.0  181377.0
+     advance/query.index.getTetraIndexTree.A-Mesh |  172770.0        1  172770.0  172770.0  172770.0
+                                        configure |      71.0        1      71.0      71.0      71.0
+                                         finalize |     704.0        1     704.0     704.0     704.0
+                                       initialize | 1333426.0        1 1333426.0 1333426.0 1333426.0
+     initialize/m2n.acceptPrimaryRankConnection.A |  810972.0        1  810972.0  810972.0  810972.0
+    initialize/m2n.acceptSecondaryRanksConnection |   47853.0        1   47853.0   47853.0   47853.0
+      initialize/m2n.broadcastVertexDistributions |       1.0        1       1.0       1.0       1.0
+             initialize/m2n.buildCommunicationMap |    5491.0        1    5491.0    5491.0    5491.0
+              initialize/m2n.createCommunications |     426.0        1     426.0     426.0     426.0
+        initialize/m2n.exchangeVertexDistribution |   41864.0        1   41864.0   41864.0   41864.0
+          initialize/partition.prepareMesh.B-Mesh |      36.0        1      36.0      36.0      36.0
+    initialize/partition.receiveGlobalMesh.A-Mesh |  468944.0        1  468944.0  468944.0  468944.0
+                     initialize/preprocess.B-Mesh |      26.0        1      26.0      26.0      26.0
+                                   solver.advance |    2278.0        1    2278.0    2278.0    2278.0
+                       solver.initialize.postinit |     178.0        1     178.0     178.0     178.0
+                        solver.initialize.preinit |     925.0        1     925.0     925.0     925.0
+```
 
-Noteworthy events are
+</div>
 
-1. The communication buildup, which can become very expensive on clusters. This is generally bound to the filesystem. This is not a problem for The main latency here is the distributed file-system.
+The output for parallel solvers is slightly more complex.
+After the name of the event, the table contains three block, each containing the sum, count, mean, min, and max runtime for a specific rank.
+
+1. the first block displays the primary rank (0)
+2. the second block displays the secondary rank which spend the least total time in `advance`
+3. the third block displays the secondary rank which spend the most total time in `advance`
+
+<div style="display:block;overflow-x:auto" markdown="1">
+
+```console
+$ precice-events analyze B
+Reading events file events.json
+Output timing are in us.
+Selection contains the primary rank 0, the cheapest secondary rank 1, and the most expensive secondary rank 1.
```

Comment:
Cheapest and most expensive ranks are identical in this example.

Reply:
Replaced it with an ASTE run